src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -46,29 +50,36 @@ public function process(ContainerBuilder $container)
         }
 
         $definition = $container->getDefinition($this->formExtensionService);
```

Comment:
IMHO `form.extension` should be a reserved service, required for the `FormPass` to do the job, so an exception should be thrown in the case it is defined but not an instance of `DependencyInjectionExtension`.

What do you think __@symfony/deciders__?